cartography/data/jobs/analysis/gcp_compute_asset_inet_exposure.json

```diff
@@ -1,41 +0,0 @@
-{
```

Comment:
Did we delete it by mistake?

I thought, this file is used to calculate public facing for few resources.